test/test_bksf_mapping.py

```diff
@@ -15,8 +15,7 @@
 import unittest
 from test import QiskitNatureTestCase
 import numpy as np
-from qiskit.quantum_info import Pauli
-from qiskit.aqua.operators import WeightedPauliOperator
+from qiskit.opflow import PauliSumOp
 from qiskit_nature.bksf import edge_operator_aij, edge_operator_bi
 
```

Comment:
Since I proposed to remove `qiskit_nature.bksf`, you can also remove this test file